### DISCUSSION TITLE
Displaying unit test results and code quality reports in graphical representation.

### DISCUSSION THREAD

**Discussion by EsukapalliReddy** [community/community#30554] [Actions]
Hi,
I've been looking for marketplace Actions to displaying unit test results and code quality reports in graphical representation like below.

![image](https://user-images.githubusercontent.com/85504813/186198462-af3ac299-5f2e-4358-a576-c70038a6a431.png)

and so far I cannot find them in GitHub marketplace.

Please suggest me how to display the html reports in GitHub Actions.

Thanks,
Jagan.

**Comment by Daizygod** [comment]
maybe this is what you search

https://github.com/allure-framework/allure2

![image](https://user-images.githubusercontent.com/46899953/186218514-f1c196fd-b7d2-4364-9bc1-32cd801e5a49.png)

**Comment by airtower-luna** [comment]
If you want something that works within the Actions UI, look at [job summaries](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#adding-a-job-summary). Maybe you can find an action to create a Markdown summary from the output of your test/review tools?
